Controller/Adminhtml/Hub/Index.php

```diff
@@ -34,6 +33,12 @@ public function __construct(
         $this->cacheManager = $cacheManager;
         $this->requestObject = $request;
 
+        $objectManager = ObjectManager::getInstance();
+
+        $this->storeManager = $objectManager->get(
+            \Magento\Store\Model\StoreManagerInterface::class
```

Comment:
Parece melhor injetar a `StoreManagerInterface` no constructor, assim como já tem outras dependências injetadas nele. 

Outras classes do projeto também já injetam a `StoreManagerInterface` dessa forma, como a `Model\CardsRepository`, precisando usar somente esse código:

`
 $this->storeManager = $storeManager;
`

Aproveitando já recomendo declarar essa e as demais classes dos parâmetros do constructor no `use` para não precisar usar o namespace completo e ficar mais limpo :)

Reply:
Feito!